cmd/mimir/help-all.txt.tmpl

```diff
@@ -481,6 +481,8 @@ Usage of ./cmd/mimir/mimir:
     	Username to use when connecting to Redis.
   -blocks-storage.bucket-store.index-cache.redis.write-timeout duration
     	Client write timeout. (default 3s)
+  -blocks-storage.bucket-store.index-header-eager-loading-startup-enabled
+    	[experimental] If enabled, store-gateway will periodically persist block IDs of lazy loaded index-headers and load them eagerly during startup. This config has no effect if index-header lazy loading is disabled.
```

Comment:
```suggestion
    	[experimental] If enabled, store-gateway will periodically persist block IDs of lazy loaded index-headers and load them eagerly during startup. This configuration has no effect if index-header lazy loading is disabled.
```
config if it’s a file; configuration is it is general language

Reply:
Updated as suggested.